tests/sentry/api/endpoints/test_project_templates_index.py

```diff
@@ -55,3 +58,70 @@ def test_get__no_access(self):
         # Ensure this errors with 403, as the user does not have access to the organization
         response = self.get_error_response(org_two.id, status_code=403)
         assert response.status_code == 403
+
+
+class ProjectTemplateIndexPostTest(APITestCase):
+    endpoint = "sentry-api-0-organization-project-templates"
+
+    def setUp(self):
+        super().setUp()
+        self.user = self.create_user()
+        self.organization = self.create_organization(owner=self.user)
+        self.team = self.create_team(members=[self.user])
+
+        self.login_as(self.user)
+
+        self.project_template = self.create_project_template(organization=self.organization)
+        self.project_template_two = self.create_project_template(organization=self.organization)
```

Comment:
Since this is the same as the other test class could you put the setup in a base that they both inherit from?